tests/stylesheets/__snapshots__/unused-modules.test.ts.snap

```diff
@@ -3,15 +3,6 @@
 exports[`unused-modules - esbuild should create valid stylesheet 1`] = `
 "body {
   background-color: #e6e6fa;
-  border: 5px solid #000;
-}
-.fbihc60 {
-  display: flex;
-}
-._36o3zb0 {
-  background: green;
-  height: 100px;
-  width: 100px;
```

Comment:
after upgrading esbuild & running tests, not sure why the new snapshot removes these CSS lines in particular

Reply:
We discussed this at the maintainers catch up this morning. While this is not ideal, it captures the differences between how the different bundlers handle tree-shaking of CSS.

It looks like esbuild is producing the incorrect result here, but there is nothing that can be done from the plugins side to avoid that.